background_scripts/commands.coffee

```diff
@@ -67,6 +67,11 @@ Commands =
           when "unmapAll"
             @keyToCommandRegistry = {}
 
+    # Push the key mapping for passNextKey into Settings so that it's available in the front end for insert
+    # mode.
+    Settings.set "passNextKeyKeys",
```

Comment:
On the other hand, it is a (fairly) static value, a function of the user's settings, and this approach requires no extra plumbing to maintain.